perfkitbenchmarker/linux_benchmarks/dpb_wordcount_benchmark.py

```diff
@@ -24,13 +24,12 @@
 
 dpb_wordcount_out_base: The output directory to capture the word count results
 
-For dataflow jobs, please build the dpb_dataflow_jar based on
+For dataflow jobs, please build the dpb_job_jarfile based on
 https://cloud.google.com/dataflow/docs/quickstarts/quickstart-java-maven
```

Comment:
Is this a manual step that has to be done before running this benchmark?

Reply:
Yes. This is a manual step, till the Dataflow SDK/Beam team can provide prebuilt job jars, similar to the artifacts that ship with Spark/Hadoop releases.